### ISSUE TITLE
bring back sort by oldest youtube

### ISSUE THREAD

**Issue by pranjal779** [Anarios/return-youtube-dislike#823]
### Extension or Userscript?

Both

### Request or suggest a new feature!

Can we bring back the sort by oldest feature in youtube
if possible
Thanks

### Can you work on this?

- [ ] Yes
- [ ] No

### Will you be available for follow-up questions to help developers implement this?

Yes

**Comment by BlohoJo** [comment]
I and I'm sure many other people want something like this. It's too bad we have to have these add-ons at all. Youtube should not be removing the most basic and useful features of the site.

I expect (for real) that Youtube is going to remove public view counts as well, in the near future.

**Comment by 3limin4t0r** [comment]
Although this suggestion might be useful. I feel like "return-youtube-dislike" is not the place. The name of the extension suggests that it does 1 thing (return dislikes), and it should do that 1 thing well. It shouldn't try to do more.

**Comment by BlohoJo** [comment]
Well then, a new add-on.

**Comment by pranjal779** [comment]
> 

yup, a new issue or a new project will do, a new add-on for return sorting to the youtube?
